nyxtools/isara_robot.py

```diff
@@ -9,7 +10,7 @@
 
 
 class IsaraRobotDevice(Device):
-    class Tool(Enum):
+    class Tool(int, Enum):  # int enum is necessary in python 3.11+?
```

Comment:
For some reason it started causing errors, even though it had been doing fine as just Enum.

Reply:
you haven't been using a newer Python to run this, have you? (not sure there would be a good reason to even try that) We have a similar Enum in config_params and it hasn't caused any similar issues before.

Reply:
@mskinner5278 Can you clarify "some problems"?

Reply:
I could not find where I wrote the "some problems" commit or comment, so I am not sure what you need cleared up. 

As for this change, one morning a user tried to begin collection and they got errors related to the typing on the enum. I put this in so that they could collect data. I could not find a reason why it suddenly decided to be an issue, and I think nyxtools just lives in the same lsdc server environment as always. The only guess is that that wasn't run properly somehow, or it was maybe updated incorrectly.

Reply:
I was refering to https://github.com/NSLS-II-NYX/nyxtools/pull/25#discussion_r1264039657